<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InitSelectAction</class>
 <widget class="QWidget" name="InitSelectAction">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="1">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Votre base de données Tavrisu est vide. Que souhaitez-vous faire ?</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QRadioButton" name="radioNothing">
     <property name="text">
      <string>Rien: je veux tout de suite utiliser Tavrisu, je remplirai ma base de
 données manuellement</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <spacer name="spacerV2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1">
    <widget class="QRadioButton" name="radioImport">
     <property name="text">
      <string>Importer une sauvegarde</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QPushButton" name="buttonNext">
     <property name="text">
      <string>Suivant &gt;</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <spacer name="spacerV1">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="1">
    <widget class="QRadioButton" name="radioSearchFolder">
     <property name="text">
      <string>Analyser mon dossier Musique pour constituer automatiquement ma
base de données</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <spacer name="spacerH2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0">
    <spacer name="spacerH1">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
